README.md

```diff
@@ -14,6 +14,8 @@
 
 # Splunk Distribution of OpenTelemetry for Node.js
 
+## **Documentation for the current stable version (1.x) can be viewed [here](https://github.com/signalfx/splunk-otel-js/tree/1.x).**
```

Comment:
```suggestion
## **Documentation for the current stable version (1.x) can be viewed [here](https://quickdraw.splunk.com/redirect/?product=Observability&location=nodejs.application&version=current).**
```

Let's use the official docs link. Unless you want to link to 1.x internal dev docs, but we'd have a separate note for that.

Reply:
Sure, do you think it makes sense to link to the 1.x GitHub branch or not? Wanted to avoid confusion when someone stumbles upon this repository and e.g. Jaeger has been removed or metrics examples are different

Reply:
And by link I mean in addition to the URL you added